compiler/prelude/prelude.go

```diff
@@ -1,10 +1,10 @@
+// Prelude is the GopherJS JavaScript interop layer that is behind
+// the github.com/gopherjs/gopherjs/js package
+const Prelude = prelude + numeric + types + goroutines + jsmapping
```

Comment:
(I thought I've already mentioned but just in case)

Could we have another PR to move this part so that we could confirm that the generated min codes are same?

Reply:
I'm confused: #794 moves the minimized js?

Reply:
@hajimehoshi I've updated the issue description in https://github.com/gopherjs/gopherjs/issues/793 to explain how the PRs are linked.

Reply:
OK so does #794 do moving the minimized code and reformatting the original js at the same time?

Reply:
Not sure what you mean by "moving"? There are no changes in terms of target files in #794, per https://github.com/gopherjs/gopherjs/issues/793

Reply:
That move happens in #784, per the summary at the bottom of #793

Reply:
Hm, #784 also changes the way of generating the minimized code, right?

Reply:
Correct

Reply:
Yeah, so my point is:

> Could we have another PR to move this part so that we could confirm that the generated min codes are same?

In #784, it is hard to say the generated codes are same, right?

Reply:
We're going to get a diff introduced by switching to use `uglify-es` at some point, simply because it's a different program/environment minifying the code with nothing that guarantees the same behaviour. So I'm not entirely convinced of the benefit of having a separate PR that moves the line of minified code from `prelude.go` to `prelude_min.go`. 

To my mind the things that matter here are:

1. the tests all pass (they intentionally run with minified code)
2. the size of the minified prelude is roughly the same:

```
$ git rev-parse HEAD
e14987c0ef06db387b90fec85e8d06dc05598e24
$ goexec -quiet 'io.WriteString(os.Stdout, prelude.Minified)' | wc -c
32740
$ git checkout regen_min
$ goexec -quiet 'io.WriteString(os.Stdout, prelude.Minified)' | wc -c
32767
```

Does that sound a reasonable compromise to you?

I've just pushed up another commit that ensures we follow the exact same convention on comments for what it's worth.

Reply:
Yeah, fair enough. Thank you!